### ISSUE TITLE
negative evar in nsnmf and pmf.

### ISSUE THREAD

**Issue by ccshao** [marinkaz/nimfa#19]
Negative evar in nsnmf an pmf methods (version 1.2.1), with  huge rss;

One example for nsnmf:
import nimfa
import numpy as np
V = np.random.rand(40, 100)
nsnmf = nimfa.Nsnmf(V, seed="random", rank=10, max_iter=12, theta=0.5)
nsnmf_fit = nsnmf()
nsnmf_fit.fit.evar()
-859782115351077.0
nsnmf_fit.fit.rss()
1.1591715624174236e+18

One example for Pmf:
V = nimfa.examples.medulloblastoma.read(normalize=True)
pmf = nimfa.Pmf(V, seed='nndsvd', rank=10, max_iter=100)
pmf_fit = pmf()
pmf_fit.fit.evar()
-875.87260969308397

**Comment by marinkaz** [comment]
Fixed in 4d6a6951f06f45b445694d688a7653d94036d77d.

**Comment by marinkaz** [comment]
The PMF on the medulloblastoma data should produce Evar similar to:

```python
import nimfa
V = nimfa.examples.medulloblastoma.read(normalize=True)
pmf = nimfa.Pmf(V, seed='nndsvd', rank=10, max_iter=100)
pmf_fit = pmf()
print(pmf_fit.fit.evar())

>>> 0.906489
```
